lib/cgi/cookie.rb

```diff
@@ -123,6 +132,14 @@ def secure=(val)
       @secure
     end
 
+    # Set whether the Cookie is a httponly cookie or not.
+    #
+    # +val+ must be a boolean.
+    def httponly=(val)
+      @httponly = val if val == true or val == false
```

Comment:
I think '=!!val' is better, very simple. 
https://github.com/xibbar/ruby/commit/caac34fb6a9f22367ccea09d97221b0ae2202d99

Do you think this change? @knu